### DISCUSSION TITLE
Display hidden beatmaps' difficulty icon at song select greyed out

### DISCUSSION THREAD

**Discussion by nihodi** [ppy/osu#20689] [Ideas]
Currently, there is no easy way to tell if a beatmapset has hidden difficulties at a glance. You have to right-click and see if there is a "Restore all hidden" option. This would make it way easier to tell if there are any hidden difficulties.

|                Before       |               After           |
|-----------------------|-----------------------|
|![Screenshot_20221010_163037](https://user-images.githubusercontent.com/100531962/194890756-1a56dd60-7178-4a4d-8845-d460297a80d8.png)|![Screenshot_20221010_163128](https://user-images.githubusercontent.com/100531962/194890834-aac99c34-793e-4dc8-b726-ed86f6efe3e2.png)|

**Comment by peppy** [comment]
I don't think making it grey is the best choice, as it may look like a higher difficulty. But I think displaying in some fashion may be good. Needs some experimentation.

**Reply by ghost** [reply]
pretty sure it already greys out when theres certain difficulties that match a search filter but other difficulties don't (e.g. selecting mania as the ruleset will display catch difficulties as greyed out)

**Reply by peppy** [reply]
Right. Might be okay then? Although I get the feeling some people would prefer hidden beatmaps to not display in this area at all. Not sure.
